smartdashboard/Experiment_Overview.py

```diff
@@ -55,9 +56,15 @@ def build_app(manifest_path: str) -> None:
         error_builder(ex)
     else:
         views = overview_builder(manifest)
+        to_update: t.Iterable[EntityView[t.Any]] = (
+            views.exp_view,
+            views.app_view,
+            views.orc_view,
+            views.ens_view,
+        )
 
         while True:
-            for v in [views.exp_view, views.app_view, views.orc_view, views.ens_view]:
+            for v in to_update:
```

Comment:
Would a mixin like `class LiveView` enable us to avoid hard-coding this `to_update` list and reduce the amount of changes we do in the future for each new view?

```py
class ExperimentView(EntityView, LiveView):...
...
```
would let `build_app` do something like:

```py
to_update = [v for v in view if isinstance(v, LiveView)]
```

It's potentially too early to worry about it... @MattToast

Reply:
Upon review, I think `EntityView` actually does what i'm asking, already.

`to_update = [v for v in views if isinstance(v, EntityView)]`

Reply:
Can this be resolved then?

Reply:
the same core exists: do you want to avoid manually building the to_update list. if it's not a big deal, feel free to close

Reply:
closing, but i will keep this in mind as the dashboard grows!